spec/basket/backend_adapter/redis_backend_spec.rb

```diff
@@ -2,20 +2,20 @@
   describe "#data" do
     it "returns all the basket entries" do
       backend = described_class.new
-      backend.push("test_queue_1", {a: 1})
-      backend.push("test_queue_1", {a: 2})
-      backend.push("test_queue_2", {b: 1})
+      backend.push("test_queue_1", {"a" => 1})
```

Comment:
`JSON.parse` always returns string keys since that's the JSON format. 

However, if we wanted more flexibility we could leverage ActiveSupport's `hash__with_indifferent_access`, if we wanted to included that dependency.

Reply:
I like this, no need to bring in ActiveSupport if we don't want it yet.